张依涵/10.27

```diff
@@ -0,0 +1,31 @@
+#include <studio.h>
+int main()
+{   int a[10000],n,m,k,i,j,z;
```

Comment:
Change your code like 
int main()
{
    int a[10000], n, m, k, i, j, z.
And you can define variables in for loop like
for(int i = 0; i < n; i++)
{
}
You should learn something about global variable and local variable, figure out them scope and duration